verifier/src/main/kotlin/tech/figure/classification/asset/verifier/config/VerifierClientConfig.kt

```diff
@@ -45,7 +46,7 @@ class VerifierClientConfig private constructor(
     val eventChannel: Channel<VerifierEvent>,
     val eventDelegator: AssetClassificationEventDelegator,
     val eventProcessors: Map<String, suspend (VerifierEvent) -> Unit>,
-    val eventStreamProvider: EventStreamProvider,
+    val eventStreamProvider: EventStreamProvider<Any>,
```

Comment:
Nit. I wouldn't use `Any`. This could be one of two options; block-api or query-node.
```
sealed interface BlockDto
data class BlockDataFromQueryNode(val blockData: BlockData) : BlockDto
data class BlockResultFromApi(val blockResult: BlockServiceOuterClass.BlockResult) : BlockDto
```

Reply:
i like it. Probably leave the interface unsealed for consumers to implement their own, however.